backend/api/serializers/computetask.py

```diff
@@ -106,6 +108,56 @@ def get_value(self, task_output):
             return data
 
 
+class ComputeTaskInputAssetSerializer(serializers.ModelSerializer, SafeSerializerMixin):
+    identifier = serializers.SerializerMethodField(source="*", read_only=True)
+    asset = serializers.SerializerMethodField(source="*", read_only=True)
+
+    class Meta:
+        model = ComputeTaskInputAsset
+        fields = [
+            "identifier",
```

Comment:
Would it make sense to also expose asset kind? How will the consumers know how to infer asset kind otherwise?

Reply:
Done in https://github.com/Substra/substra-backend/pull/496/commits/01130181a84b2ad9955f93faef6a5c3ed63f0f68